physics/rigid_motion.hpp

```diff
@@ -36,13 +36,12 @@ using quantities::si::Radian;
 template<typename FromFrame, typename ToFrame>
 class RigidMotion final {
  public:
-  static_assert(!std::is_same_v<FromFrame, ToFrame>,
-                "FromFrame and ToFrame must be different");
-
   RigidMotion(
```

Comment:
Same enable_if on this one (and probably a typename = void on the other one for disambiguation).

Reply:
No can do.